src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -797,6 +797,17 @@ function fs.complete(sPath, sLocation, bIncludeFiles, bIncludeDirs)
     return tEmpty
 end
 
+function fs.isMountPoint(sPath)
+    expect(1, sPath, "string")
+    if not fs.isDir(sPath) then
+        return false
+    elseif fs.combine(sPath, "..") == ".." or fs.getDrive(sPath) == fs.getDrive(fs.combine(sPath, "..")) then -- the == ".." check prevents errors when path is "/"
```

Comment:
This might be cleaner as:

```lua
local parent = fs.getDir(path)
return parent == ".." or fs.getDrive(path) ~= fs.getDrive(parent)
```

Hrmr, question: do we want to count `/` as a mount point? I'd argue yes.